pom.xml

```diff
@@ -98,6 +93,12 @@
                     <compilerArgs>
                         <arg>-parameters</arg>
                     </compilerArgs>
+                    <annotationProcessorPaths>
+                        <path>
+                            <groupId>org.hibernate.orm</groupId>
+                            <artifactId>hibernate-jpamodelgen</artifactId>
+                        </path>
+                    </annotationProcessorPaths>
```

Comment:
Note hibernate-jpamodelgen is managed by Quarkus, and dependency management is supported in `annotationProcessorPaths` starting with `maven-compiler-plugin`... 3.12 I think?